borgmatic/config/schema.yaml

```diff
@@ -1602,6 +1602,89 @@ properties:
                 example:
                     - start
                     - finish
+    zabbix:
+        type: object
+        additionalProperties: false
+        properties:
+            itemid:
+                type: integer
+                description: |
+                    The ID of the Zabbix item used for collecting data.
+                    Unique across the entire Zabbix system.
+                example: 55105
+            host:
+                type: string
+                description: |
+                    Host name where the item is stored.
+                    Required if "itemid" is not set.
+                example: borg-server
+            key:
+                type: string
+                description: |
+                    Key of the host where the item is stored.
+                    Required if "itemid" is not set.
+                example: borg.status
+            server:
+                type: string
+                description: |
+                    The address of your zabbix instance.
+                    Not specifying a server will default to the Zabbix cloud.
+                example: https://zabbix.your-domain.com
+            username:
+                type: string
+                description: |
+                    The username used for authentication.
+                example: testuser
+            password:
+                type: string
+                description: |
+                    The password used for authentication.
+                example: fakepassword
+            api_key:
+                type: string
+                description: |
+                    The API key used for authentication.
+                example: fakekey
```

Comment:
Might be good to document here the relationship between these options, just like you did with the previous ones.

Reply:
https://github.com/borgmatic-collective/borgmatic/commit/d11c517f675cf35894b33432d22cdbd9e6d36302